### ISSUE TITLE
Pi pass Pi 3

### ISSUE THREAD

**Issue by Vspin99** [Matthew-Hsu/PiPass#87]
I have had this working but now it boots and then goes to the log on screen.
Goes into starting 2 times then to log in and sits
Have reflashed the SD card and same result.
Help please

**Comment by person65278** [comment]
This is a month old but I was able to get this working on a rpi3 from a fresh install.  would you like an image of my install? I did it with a fresh raspbian install then used his install scrypt. then you have to point apache 2 to the correct spot. and I modified my piPass.py to correct other issues.  If you can get that far in all of it here is my fixed piPass.py scrypt. place it where the original is... /opt/PiPass/piPass.py

[piPass.zip](https://github.com/Matthew-Hsu/PiPass/files/1181704/piPass.zip)

**Comment by Vspin99** [comment]
image would be great
thank you

**Comment by Vspin99** [comment]
any luck on the image?

**Comment by person65278** [comment]
I had forgot about it. but here it is. Username is pi  password is
raspberry. image should be plug and play for an rpi 3. It is made from
https://github.com/Matthew-Hsu/PiPass  I made it with a fresh raspbian
install then made a cpl of the dependencies from source....... Changed some
code so it would retry instead of crashing on some errors that by what i
saw were unavoidable at random times. sorry for the size I fixed the errors
but I have never done much when it comes to making disk images so ​
 pipassrpi3.zip
<https://drive.google.com/file/d/0B_-ruduXYrW1RGZPdGlxeTlQeFk/view?usp=drive_web>
​someone who knows how to shrink them is more then welcome to to a crack at
it and take it down in size.
